rules/Php80/Rector/Class_/ChangeResolvedClassInParticularContextForAnnotation.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Php80\Rector\Class_;
+
+use Rector\BetterPhpDocParser\PhpDoc\DoctrineAnnotationTagValueNode;
+use Rector\BetterPhpDocParser\ValueObject\PhpDoc\DoctrineAnnotation\CurlyListNode;
+use Rector\Php80\ValueObject\AnnotationToAttribute;
+
+class ChangeResolvedClassInParticularContextForAnnotation
```

Comment:
outer service should not inside `Rector` subnamespace, please move it to another places, eg:

https://github.com/rectorphp/rector-src/tree/main/packages/BetterPhpDocParser/Annotation